### PR TITLE
Added command to list all Windows instances in the account

### PR DESCRIPTION
**Pull request by jonathanwcrane**
Makes it simple to get all your Windows instances in one go.  Unfortunately, the complement is not straightforward.

### CODE REVIEW

**Comment by marcesher** [comment]
works great, @jonathanwcrane .

Please add a line at the top of the file for `help`, following the format of other commands listed there

**Comment by jonathanwcrane** [comment]
Like so?

File: scripts/ec2/ls.coffee [hunk 1]
```diff
@@ -13,6 +13,7 @@
 #   hubot ec2 filter sometext - Filters instances whose name (name tag value) contains 'sometext'
 #   hubot ec2 expired - Displays instances that have expired
 #   hubot ec2 expiring - Displays instances that are expiring within 2 days
+#   hubot ec2 windows - Displays widows instances
```
Comment (marcesher) [inline scripts/ec2/ls.coffee]: it should not display widows

Windows would be good though

**Comment by jonathanwcrane** [comment]
It's funny because I have scripts that delete orphan AMIs.  But no widows in AWS that I know of. Should be fixed now (?)

**Comment by marcesher** [comment]
thanks @jonathanwcrane !

**Pull request merged**